pkg/tr31/des.go

```diff
@@ -59,6 +60,8 @@ func bitsOn(b byte) int {
 }
 
 // Encrypt3DESCBC encrypts plaintext using 3DES in CBC mode with the provided 16-byte key.
+//
+//gosec:disable G502 -- Legacy system requires 3DES
```

Comment:
This (and the one at the bottom) aren't needed anymore - I don't think they're working anyway.